web/src/message_view_header.ts

```diff
@@ -112,6 +113,27 @@ function get_message_view_header_context(filter: Filter | undefined): MessageVie
                 }),
             };
         }
+
+        if (inbox_util.is_visible() && inbox_util.is_channel_view()) {
+            const stream_name_with_privacy_symbol_html = render_inline_decorated_channel_name({
+                stream: current_stream,
+                show_colored_icon: true,
+            });
+            context = {
+                ...context,
+                title: undefined,
+                html_title: $t_html(
+                    {defaultMessage: "All topics in&nbsp; <z-icon></z-icon>"},
```

Comment:
`z-icon` might be better called `z-channel`. Also, do we need the `nbsp` or is there a better way to manage the styling? Might be worth explaining there.